docs/src/kubectl-plugin.md

```diff
@@ -780,6 +781,39 @@ kubectl cnpg logs cluster cluster-example --output my-cluster.log
 Successfully written logs to "my-cluster.log"
 ```
 
+#### Pretty
+
+The `pretty` sub-command reads a log stream from standard input and formats it
+into a human-readable output, and attempts to sort the entries by timestamp.
+
+It is intended to be used in combination with `kubectl cnpg logs cluster`, as
+shown in the following example:
+
+```
+$ kubectl cnpg logs cluster cluster-example | kubectl cnpg logs pretty
```

Comment:
if we don't get rid of `Kubectl cnpg logs`, wouldn't `pretty` make better sense as an option?
`Kubectl cnpg logs --pretty cluster cluster-example`

Reply:
In that case, you won't be able to use it with the logs you collected before on your own.

Reply:
No. Responding here for traceability, rather than deleting.
Seeing all the options supported by `pretty`, it would be unwieldy.